### PR TITLE
add healthcheck endpoint

### PR DESCRIPTION
**Pull request by johnseekins**
Simple endpoint for load balancers to check individual instances. Lighter-weight than using `/metrics`.

### CODE REVIEW

**Comment by palmerabollo** [comment]
Thanks @johnseekins, could you please update the deployment template? https://github.com/Telefonica/prometheus-kafka-adapter/blob/master/helm/prometheus-kafka-adapter/templates/deployment.yaml#L98-L105

**Review by palmerabollo — APPROVED** [review]

**Comment by palmerabollo** [comment]
LGTM cc @jpfe-tid just in case you want to add something else. Otherwise I'll merge it in a couple of days.

File: main.go [hunk 1]
```diff
@@ -74,6 +74,7 @@ func main() {
 	r.Use(ginrus.Ginrus(logrus.StandardLogger(), time.RFC3339, true), gin.Recovery())
 
 	r.GET("/metrics", gin.WrapH(prometheus.UninstrumentedHandler()))
+	r.GET("/health", func(c *gin.Context) { c.JSON(200, gin.H{"status": "UP"}) })
```
Comment (jpfe-tid) [inline main.go:77]:
```suggestion
	r.GET("/healthz", func(c *gin.Context) { c.JSON(200, gin.H{"status": "UP"}) })
```

File: helm/prometheus-kafka-adapter/templates/deployment.yaml [hunk 2]
```diff
@@ -97,11 +97,11 @@ spec:
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: /metrics
+              path: /health
               port: http
           readinessProbe:
             httpGet:
-              path: /metrics
+              path: /health
```
Comment (jpfe-tid) [inline helm/prometheus-kafka-adapter/templates/deployment.yaml:104]:
```suggestion
              path: /healthz
```

File: helm/prometheus-kafka-adapter/templates/deployment.yaml [hunk 3]
```diff
@@ -97,11 +97,11 @@ spec:
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: /metrics
+              path: /health
```
Comment (jpfe-tid) [inline helm/prometheus-kafka-adapter/templates/deployment.yaml:100]:
```suggestion
              path: /healthz
```

**Review by jpfe-tid — APPROVED** [review]

**Pull request merged**